packages/build/src/plugins_core/functions/index.js

```diff
@@ -24,6 +24,7 @@ const normalizeFunctionConfig = ({ buildDir, functionConfig = {}, isRunningLocal
   includedFiles: functionConfig.included_files,
   includedFilesBasePath: buildDir,
   ignoredNodeModules: functionConfig.ignored_node_modules,
+  schedule: functionConfig.schedule,
```

Comment:
Should this be added to `@netlify/config` too?
  - https://github.com/netlify/build/blob/fb173c096310fc354867567eff27b4c4cabd0d8b/packages/config/src/functions_config.js#L50
  - And https://github.com/netlify/build/blob/fb173c096310fc354867567eff27b4c4cabd0d8b/packages/config/src/validate/validations.js#L220
  - Potentially also https://github.com/netlify/build/blob/fb173c096310fc354867567eff27b4c4cabd0d8b/packages/functions-utils/tests/main.js#L110